src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-field {
+  margin-top: 20px;
+}
+
+.search-icon-big {
+  position: absolute;
+  margin-top: 25px;
+  margin-left: 25px;
+}
+
+.search-bar {
+  width: 100%;
+
+  box-sizing: border-box;
+
+  border: 1px solid rgb(225, 231, 237);
+  border-radius: 4px;
+
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  font-family: Avenir, serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.big-search-bar {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.search-bar:hover {
+  border: 1px solid rgb(225, 231, 237);
```

Comment:
You have the same styles for the border in the `.search-bar` selector. So it's redundant here